irc/bitlbee/Portfile

```diff
@@ -16,10 +16,12 @@ long_description    bitlbee is an IRC server that can connect to other IM \
                     instant messaging.
 
 homepage            https://www.bitlbee.org/
-master_sites        https://get.bitlbee.org/src/
+master_sites        https://github.com/bitlbee/bitlbee/archive/refs/tags/
```

Comment:
if you want to do this you should instead use the [`github` PG](https://guide.macports.org/index.html#reference.portgroup.github) which will do all kinds of things for you automatically.